.github/workflows/tests.yaml

```diff
@@ -55,10 +55,13 @@ jobs:
                       symfony-version: '4.4.*'
                     - package: 'serialization'
                       symfony-version: '4.4.*'
-                    # Exclude it is not support on PHP 8
+                    # Exclude as it is not support on PHP 8
                     # https://github.com/php-amqplib/php-amqplib/issues/846
```

Comment:
@cmodijk What to do with this? Because https://github.com/php-amqplib/php-amqplib/issues/846 is closed...

Reply:
As long as they don't support it we can't support it so for know i would keep the ignore until they change there mind about it.